bucket/jmkvpropedit.json

```
@@ -0,0 +1,26 @@
+{
+    "version": "1.5.2",
+    "description": "A batch GUI for mkvpropedit (part of MKVToolNix) written in Java",
+    "homepage": "https://github.com/BrunoReX/jmkvpropedit",
+    "license": "BSD 2-Clause \"Simplified\" License",

```

```suggestion
    "license": "BSD-2-Clause",

```
BSD 2-Clause "Simplified" License == BSD 2-Clause
See https://spdx.org/licenses/BSD-2-Clause.html